### PR TITLE
remove almp3

### PR DESCRIPTION
**Pull request by pbomta**
also moves rest of sound related functions from common to cGame

Closes: #388

### CODE REVIEW

File: cGame.h [hunk 1]
```diff
@@ -79,7 +78,7 @@ class cGame : public cScenarioObserver, cInputObserver {
 
 	int m_pathsCreated;
 
-    int m_musicVolume;              // volume of the mp3 / midi
+    int m_musicVolume;              // volume of the music
```
Comment (pbomta) [inline cGame.h:81]: This data member is not used. Shall we leave it in for now? I believe the plan is to have volume settings in the menu at some point.

**Comment by stefanhendriks** [comment]
I feel sad to see MP3 support go, but I know why it happens. I have fond memories of having MP3 files with more adlib or MT32 sounds of the Dune 2 music. This will return once we can migrate to SDL or something. All for the better.

Nice improvements otherwise!

**Comment by stefanhendriks** [comment]
(btw, MP3 support was broken anyway, so not that sad ;-) )

File: cGame.h (hunk 1 above)
Comment (stefanhendriks) [reply cGame.h:81]: 🤔 , I guess we can keep it. Once we have a proper configuration object we can move it to there.

**Review by stefanhendriks — APPROVED** [review]

**Pull request merged**